docs/rackhd/samples/install-os-minimal-parameters.json

```diff
@@ -0,0 +1,11 @@
+{
+    "options": {
+        "defaults": {
+            "version": "7",
+            "repo": "http://172.31.128.1:9080/mirrors/centos/7/os/x86_64",
+            "rootPassword": "root12345",
+            "hostname": "rackhd-node",
```

Comment:
copy 'required' items from confluence page, maybe that page is out of date, let me have a check